docs/tsconfig.json

```diff
@@ -33,5 +33,5 @@
     "server.ts",
     "./.mdx"
   ],
-  "exclude": ["node_modules"]
+  "exclude": ["node_modules", ".contentlayer"]
```

Comment:
I am not very familiar with `Contentlayer`, but the [`Contentlayer` documentation](https://contentlayer.dev/docs/getting-started-cddd76b7#typescript-configuration) suggests including `.contentlayer/generated` in `tsconfig.json`.

Should `.contentlayer` be really excluded?

Reply:
I did it haphazardly. I'll fix it.